instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -532,12 +532,34 @@ def test_instrument_client(self):
             self.assertEqual(result.text, "Hello!")
             self.assert_span(num_spans=1)
 
+        def test_instrumentation_without_client(self):
+
+            HTTPXClientInstrumentor().instrument()
+            results = [
+                httpx.get(self.URL),
+                httpx.request("GET", self.URL),
+            ]
+            with httpx.stream("GET", self.URL) as stream:
+                stream.read()
+                results.append(stream)
+
+            spans = self.assert_span(num_spans=len(results))
+            for idx, res in enumerate(results):
```

Comment:
I would add a subtest with `with self.subTest(idx=idx):` inside the loop to ease debugging in case of failure